packages/synapse-react-client/src/style/components/_markdown.scss

```diff
@@ -80,8 +81,10 @@
     content: none;
   }
 }
-.markdown ul li {
-  padding-top: 5px;
+
+.markdown ol + p,
+.markdown ul + p {
+  margin-top: 10px;
 }
```

Comment:
There was not enough spacing for paragraphs following lists, which the agent likes to use a lot.